src/core/Entry.cpp

```diff
@@ -325,25 +325,28 @@ QString Entry::totp() const{
 
         int rc = oath_init();
```

Comment:
You should check the return value after this or not save it all and wrap the call in Q_UNUSED().